distribution/docs/src/main/resources/content/config.adoc

```diff
@@ -12,7 +12,7 @@ Version: ${project.version}. Copyright (c) Codice Foundation
 :sectnums:
 :sectnumlevels: 5
 :title-logo-image: image::alliance-logo.png[]
-:external-link: image:external-link.png[This link is outside the ${branding} documentation]
+:external-link: image::external-link.png[This link is outside the ${branding} documentation]
```

Comment:
@ricklarsen, this is the change to a doc which I made so that the tests pass. This change deviates from how the doc should look, though. I'm investigating the differences between DDF and Alliance which contribute to the fact that the tests pass for DDF in this same situation.

Reply:
I think we can come back around and fix this outside of this PR